### PR TITLE
Fix bug in Faraday

### PR DESCRIPTION
**Pull request by psousa**
Urgent. Bug occurs in latest version of Faraday (>0.9.0).
All tests pass.

### CODE REVIEW

File: invoicexpress.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.files = %w(CHANGELOG.md README.md Rakefile invoicexpress.gemspec)
   spec.files += Dir.glob("lib/**/*.rb")
   spec.files += Dir.glob("spec/**/*")
-  spec.homepage = "http://invoicexpress.pt"
+  spec.homepage = "http://www.invoicexpress.net"
```
Comment (theplastictoy) [inline invoicexpress.gemspec]: http://invoicexpress.com

**Comment by theplastictoy** [comment]
:ok:

File: invoicexpress.gemspec (hunk 1 above)
Comment (psousa) [inline invoicexpress.gemspec]: ah, oops. :P

**Comment by psousa** [comment]
Informação corrigida.

**Comment by lamuria** [comment]
:ok:

**Comment by paulosilva86** [comment]
:ok:

**Pull request merged**